crates/holochain/src/bin/holochain/main.rs

```diff
@@ -89,6 +89,14 @@ async fn async_main() {
     holochain_trace::init_fmt(opt.structured.clone()).expect("Failed to start contextual logging");
     debug!("holochain_trace initialized");
 
+    if let Some(true) = &config.disable_metrics {
+        tracing::warn!("Running without local metrics");
+    } else {
+        let mut path: std::path::PathBuf = config.environment_path.clone().into();
+        path.push("influxive");
```

Comment:
The way this is written assumes we're using Influx to output metrics before initializing metrics. Can this be moved inside the `initialize_metrics` call and be configurable like the logging is?

Reply:
mmm, good point. I'll just name this directory "metrics" and it may not be used if we are reporting remotely or some other strategy.